zubhub_frontend/zubhub/src/views/create_project/createProject.style.js

```diff
@@ -41,31 +41,76 @@ export const createProjectStyle = theme => ({
             textDecoration: 'underline',
         },
     },
+    selectModeTextContainer: {
+        display: 'flex',
+        flexDirection: 'column',
+        maxWidth: '750px',
+        textAlign: 'center',
+    },
     modeItemContainer: {
         display: 'flex',
+        justifyContent: 'space-between',
         gap: 20,
-        marginTop: 40,
+        paddingRight: 40,
+        paddingLeft: 40, 
+        marginTop: 60,
         [theme.breakpoints.down('md')]: {
-            flexDirection: 'column'
+            flexDirection: 'column',
+            alignItems: 'center'
         },
     },
     modeItemSelected: {
         border: `solid 1.5px ${colors.primary} !important`
     },
     modeItem: {
+        width: '260px',
+        height: '220px',
+        position: 'relative',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
         backgroundColor: colors.white,
         borderRadius: 8,
-        padding: '16px 24px',
+        padding: '18px 24px',
         border: `solid 1.5px transparent`,
-        width: '50%',
+        boxShadow: '2px 4px 5px 0px #00B8C41A',
         '&:hover': {
             border: `solid 1.5px ${colors.primary}`,
             cursor: 'pointer'
         },
         [theme.breakpoints.down('md')]: {
-            width: '100%'
+            width: '100%',
+            maxWidth: '500px',
         },
     },
+    modeItemDescription: {
+        textAlign: 'center',
+        fontSize: '12px',
+        fontWeight: '300'
+    },
+    modeItemIcon: {
+        color: colors.primary,
+        marginBottom: 2,
+        fontSize: 40,
+    },
+    modeSubmitButton: {
+        width: '150px',
+        height: '40px',
+        fontFamily: 'Raleway, Roboto, sans-serif',
+        fontSize: '16px',
+        fontWeight: 600,
+        lineHeight: '20px',
+        letterSpacing: 0,
+        marginTop: 60,
+        marginBottom: 60,
+        padding: 4,
+        alignSelf: 'center',
+        '&:disabled': {
+            backgroundColor: '#00B8C433',
```

Comment:
Sorry left this earlier: Can we add this color in theme and then use here? Same with below line

Reply:
ready